src/libs/ecs-lib.ts

```diff
@@ -0,0 +1,47 @@
+var _ = require("lodash");
+import {
+  ECSClient,
+  DescribeTasksCommand,
+  ListTasksCommand,
+  waitUntilTasksRunning,
+} from "@aws-sdk/client-ecs";
+
+export async function findIpForEcsService(cluster: string | undefined) {
+  const client = new ECSClient({});
+  const listStacksCommandResponse = await client.send(
+    new ListTasksCommand({
+      cluster,
+      desiredStatus: "RUNNING",
+    })
+  );
+  const taskArns = listStacksCommandResponse.taskArns;
+  if (typeof taskArns === "undefined") {
+    throw "taskArns undefined";
+  }
+  if (taskArns.length == 0) {
+    throw "taskArns empty";
+  }
+  waitUntilTasksRunning(
+    { client, maxWaitTime: 600 },
+    {
+      cluster,
+      tasks: [taskArns[0]], // TODO: figure out why this is only waiting for the first task.
```

Comment:
@mdial89f I had marked this item as a TODO.   It seems like this line could be `tasks: taskArns`.  Do you recall a reason that its not?  Do we not want the task arns after index 0?